configs/tsconfig-base.json

```diff
@@ -13,7 +13,6 @@
     "noUnusedLocals": true,
     "noUnusedParameters": true,
     "pretty": true,
-    "sourceMap": true,
```

Comment:
What was your reason for dropping this? I do very much want to keep the source maps enabled, they're invaluable for debugging.